src/predict.jl

```diff
@@ -0,0 +1,292 @@
+"""
+    StatsBase.predict!(y::AbstractVector{T}, m::LinearMixedModel{T},
+                    newX::AbstractMatrix{T}=m.X; use_re=true)
+    StatsBase.predict(m::LinearMixedModel{T},
+                    newX::AbstractMatrix{T}=m.X; use_re=true)
+    StatsBase.predict!(y::AbstractVector{T}, m::GeneralizedLinearMixedModel{T},
+                    newX::AbstractMatrix{T}=m.X; use_re=true, type=:response)
+    StatsBase.predict(m::GeneralizedLinearMixedModel{T},
+                    newX::AbstractMatrix{T}=m.X; use_re=true, type=:response)
+
+Predict response for new values of the fixed-effects matrix X.
+
+The modifying methods overwrite `y` in place, while the non modifying methods
```

Comment:
Would it be better to say `mutating` rather than `modifying`?  I think the term `mutating` is used more frequently.